SETUP.md

```diff
@@ -0,0 +1,39 @@
+# Nylas Use Case Samples
+
+**⚠️ Warning: This repo is not meant for use in production and stability is not fully guaranteed.**
+
+Explore working front-end and back-end sample code using an integration with Nylas.
+
+The sample application redirects to a prebuilt authentication page hosted on Nylas, then allows that authenticated user to interact with Nylas API from a web client.
+
+## ⚙️ Environment Setup
+
+Let’s check that our environment is set up to use the [Nylas Node SDK](https://github.com/nylas/nylas-nodejs). Check the Node version in your terminal:
+
+```bash
+$ node -v
+v18.0.0
+```
+
+If you don’t see a version returned, you may not have Node installed. Try the following steps:
+
+1. Visit [nodejs.org](https://nodejs.org/en/) to set up Node on your machine
+2. _Recommended_: If you happen to use or require multiple versions of Node, consider using [nvm](https://github.com/nvm-sh/nvm)
+
+The minimum required Node version is `v18.0.0`. As a quick check, try running `node -v` again to confirm the version. You may need to restart your terminal for the changes to take effect.
+
+## ⚡️ App Setup
+
+1. Installation
+
+   ```bash
+   npm install
+   ```
+
+2. Run code - starts server + opens browser to `localhost:3000`
+
+   ```bash
+   npm run start
+
+   # terminal shows webhook output here
+   ```
```

Comment:
I think the code comment may be confusing, so I think keeping it to just the npm command would be best. What do you think?
```suggestion
   ```bash
   npm run start
   ```
```